### PR TITLE
Feature / About page

### PR DESCRIPTION
**Pull request by adlamas**
### Description
Link to about page

### Trello card
[Card](https://trello.com/c/ploENk37/18-2-as-a-user-i-should-be-able-to-check-the-about-section-so-i-can-learn-more-about-target-mvd)

Captura de Pantalla 2023-09-14 a la(s) 16 28 38

### CODE REVIEW

**Review by joaquincollado — APPROVED** [review]

File: src/components/About.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+import AboutPanel from 'components/home/homeScreen/about'
+import HamburgerMenu from './home/common/hamburgerMenu/hamburgerMenu';
+import RightSection from './common/RightSection';
+
+const About = () => {
+  return (
```
Comment (jpirazusta) [inline src/components/About.tsx:7]: You can implicitly return here:
```suggestion
const About = () => (
```

File: src/components/About.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <div className="App">
+      <header className="App-header">
+      </header>
```
Comment (jpirazusta) [inline src/components/About.tsx:10]:
```suggestion
      <header className="App-header" />
```

File: src/components/About.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      <body>
+        <section className="main-section">
+          <section className="main-section--left-side">
+            <HamburgerMenu />
+            <div className="input-section">
+              <AboutPanel />
+            </div>
+          </section>
+              <RightSection />
```
Comment (jpirazusta) [inline src/components/About.tsx:19]: Remember to add a code formatter

File: src/components/home/common/hamburgerMenu/hamburgerMenu.css [hunk 4]
```diff
@@ -0,0 +1,27 @@
+
+.hamburguer-menu {
+  position: fixed;
+  margin: 2em 0 0 2em;
+}
+
+.menu-section {
+  display: flex;
+  flex-direction: row;
```
Comment (jpirazusta) [inline src/components/home/common/hamburgerMenu/hamburgerMenu.css:9]: `row` is the default behavior when using `flex`
```suggestion
```

File: src/components/home/common/hamburgerMenu/hamburgerMenu.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import 'components/home/common/hamburgerMenu/hamburgerMenu.css'
+import menu from 'assets/images/menu.png' 
+import { useState } from 'react';
+
+const HamburguerMenu = () => {
+  const [isMenuOpen, setMenuOpen] = useState(false);
+  const [showAboutLink, setShowAboutLink] = useState(false);
+
+  const toggleMenu = () => {
+    setMenuOpen(!isMenuOpen);
```
Comment (jpirazusta) [inline src/components/home/common/hamburgerMenu/hamburgerMenu.tsx:10]: It's good practice to use the previous state value you get from the callback, rather that using manually the value
```suggestion
    setMenuOpen(prevState => !prevState);
```

File: src/components/home/homeScreen/about.tsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
+import smiliesIcon from 'assets/images/smilies.png'
+
+import 'components/home/homeScreen/about.css'
+
+const about = () => {
```
Comment (jpirazusta) [inline src/components/home/homeScreen/about.tsx:5]: The convention for naming components is to use capital
```suggestion
const About = () => {
```

**Review by jpirazusta — APPROVED** [review]

File: src/components/About.tsx (hunk 3 above)
Comment (adlamas) [reply src/components/About.tsx:19]: Yup. I added it here https://github.com/adlamas/react-target-ts/pull/20 😄

**Pull request merged**